src/test/unittest/ut_setup.c

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2020, Intel Corporation */
+
+/*
+ * ut_setup.h -- utility helper functions for libpmem2 tests setup
+ */
+#include "../../libpmem2/config.h"
+#include "ut_fh.h"
+#include "ut_pmem2.h"
+#include "ut_setup.h"
+#include "unittest.h"
+
+/*
+ * ut_prepare_config -- fill pmem2_config, this function can not set
+ * the wrong value
+ */
+void
+ut_prepare_config_internal(struct pmem2_config *cfg, struct pmem2_source **src,
```

Comment:
I don't like this name.

Reply:
I propose names: ut_prepare_config (use non public API) and ut_prepare_config_integration (use only public API.)